lib/openxml/pptx/version.rb

```diff
@@ -1,5 +1,5 @@
 module OpenXml
   module Pptx
-    VERSION = "0.2.0"
+    VERSION = "0.2.1"
```

Comment:
🤔 Is this a patch or a minor version change?

Reply:
It's so minor, I was considering it a patch. 🤷‍♂ (technically, it _is_ solving a bug, though I suppose it could easily be construed as adding a new feature)

Reply:
It's a big deal. I just thought it might be worth bringing up. 👍